CGPs/cgp-0040/cgp_0040.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "contract": "Exchange",
```

Comment:
this should be an address I think, the tooling doesn't resolve this, right @yorhodes?

Reply:
I think the name should resolve, worked well with CGP 33 & 34

Reply:
`celocli governance:propose` translates these human readable representations to the environment specific calldata using verified ABIs and the Registry deployed at `0x0...ce10`

Reply:
@martinvol It might be helpful having the JSON file inside the CGP so that folks can read it on-chain. What do you think?

Reply:
To me is usefull, because after it executes is removed on-chain.